events-api/src/modules/events/events.service.ts

```diff
@@ -1,10 +1,66 @@
 import { EventModel } from "#/modules/events/events.model";
 import { EventsRepository } from "#/modules/events/events.repository";
+import { TicketModel } from "../tickets/tickets.model";
+import { eventsCreateRequestBodyDto } from "./dto/requests/events-create-request-body.dto";
+import { eventsUpdateRequestBodyDto } from "./dto/requests/events-update-request-body.dto";
 
 export class EventsService {
   static async getEvents(): Promise<EventModel[]> {
     const events = await EventsRepository.getAll();
 
     return events;
   }
+
+  static async createEvent(
+    data: eventsCreateRequestBodyDto,
+  ): Promise<EventModel[]> {
```

Comment:
Why to return a list for a singular method?

Reply:
fixed